plugins/UM3NetworkPrinting/resources/qml/ConfigurationChangeBlock.qml

```diff
@@ -0,0 +1,243 @@
+import QtQuick 2.2
+import QtQuick.Dialogs 1.1
+import QtQuick.Controls 2.0
+import QtQuick.Controls.Styles 1.4
+import QtGraphicalEffects 1.0
+import QtQuick.Layouts 1.1
+import QtQuick.Dialogs 1.1
+import UM 1.3 as UM
+
+Rectangle {
```

Comment:
QML wise it's better to use Item as the root object. That way you can control it better (eg; right now someone can make a ConfigurationChangeBlock and set its color property (since it's root is a rectangle).
Even if that's what you want, it's better to have a root item that aliases the color property of the rectangle.

Reply:
These have been removed.